src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -503,6 +504,8 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "BiasGelu", 1, bias_gelu);
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "EmbedLayerNormalization", 1, embed_layer_normalization);
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "SkipLayerNormalization", 1, skip_layer_normalization);
+
+    REGISTER_OPERATOR("LayerNormalization", 1, layer_norm);
```

Comment:
```suggestion
    REGISTER_OPERATOR("LayerNormalization", 1, layer_normalization);
```

Also please move this call to around the line 357 and maintain the alphabetic order.